docs/en/common_usage/set_random_seed.md

```diff
@@ -0,0 +1,39 @@
+# How to Set Random Seed
+
+As described in [PyTorch REPRODUCIBILITY](https://pytorch.org/docs/stable/notes/randomness.html), there are  2 factors affecting the reproducibility of an experiment, namely random number and nondeterministic algorithms.
+
+MMEngine provides the ability to set the random number and select a deterministic algorithm. Users can simply set the `randomness` argument of the `Runner`. The argument is eventually consumed in [set_random_seed](mmengine.runner.set_random_seed) and it has the following three fields:
+
+- seed (int): The random seed. If this argument is not set, a random number will be used.
+- diff_rank_seed (bool): Whether to set different seeds for different processes by adding the `rank` (process index) to the seed.
+- deterministic (bool): Whether to set deterministic options for the CUDNN backend.
+
+Let's take the [Get Started in 15 Minutes](../get_started/15_minutes.md) as an example to demonstrate how to set `randomness` in MMEngine.
+
+```python
+runner = Runner(
+    model=MMResNet50(),
+    work_dir='./work_dir',
+    train_dataloader=train_dataloader,
+    optim_wrapper=dict(optimizer=dict(type=SGD, lr=0.001, momentum=0.9)),
+    train_cfg=dict(by_epoch=True, max_epochs=5, val_interval=1),
+    val_dataloader=val_dataloader,
+    val_cfg=dict(),
+    val_evaluator=dict(type=Accuracy),
+    # adding randomness setting
+    randomness=dict(seed=0),
+)
+runner.train()
+```
+
+However, there may still be some differences between any two experiments, even with the random number set and the deterministic algorithms chosen. Here we briefly explain why and where these fluctuations come from.
+
+First, randomness not only exists in OpenMMLab algorithms like MMDetection. You may encounter it in most of the deep learning frameworks. The core reason is that the atomic operations in CUDA are unordered and random during parallel training.
```

Comment:
Suggest only translating this part to english:

![image](https://user-images.githubusercontent.com/57566630/222098913-1c4f45a3-eeae-45a6-ab29-cf5b108f1f33.png)

and maintain the coherence between contexts.

Reply:
Plz check my latest commit.